File: app/models/choice.py
from django.db import models
from app.models.question import Question


class Choice(models.Model):
    def __str__(self):
        return (self.choice_text[:60] + '..') if len(self.choice_text) > 60 else self.choice_text

    choice_id = models.BigAutoField(primary_key=True)
    choice_text = models.TextField()
    is_correct = models.BooleanField(default=False)
    question = models.ForeignKey(Question, on_delete=models.CASCADE)

File: express/models/Choice.js
const SQLModel = require('sequelize').Model;

module.exports = (sequelize, DataTypes) => {
  class Choice extends SQLModel {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
  }
  Choice.init(
    {
      choice_id: {
        type: DataTypes.INTEGER,
        primaryKey: true
      },
      choice_text: DataTypes.STRING,
      is_correct: DataTypes.BOOLEAN,
      question_id: DataTypes.INTEGER,
    },
    {
      sequelize,
      modelName: 'Choice',
      tableName: 'final_choice',
      freezeTableName: true,
      timestamps: false,
    },
  );
  return Choice;
};
